google_guest_agent/addresses.go

```diff
@@ -143,6 +143,9 @@ func getLocalRoutes(ifname string) ([]string, error) {
 		line = strings.TrimPrefix(line, "local ")
 		line = strings.TrimSpace(line)
 		if line != "" {
+			if !strings.Contains(line, "/") {
```

Comment:
Is the caller of `getLocalRoutes` expecting to get IP addresses, or IP address blocks?

Reply:
this function returns what the 'ip route' command outputs. if a local route has any mask other than /32, it will be present in the output.